storybook/global-a11y-rules.ts

```diff
@@ -25,4 +25,10 @@ export const globalA11yRules = [
     id: "aria-allowed-role",
     selector: ':not(button[name="day"][role="gridcell"])',
   },
+  {
+    // In some edgecases like ToggleTip we want to be able to disable the rule and hide the button from screen readers
```

Comment:
If you care: this could update to reference react aria Select's usage instead, in case someone comes along and thinks they can delete this now